src/plugins/azgenerator/CodeModelAzImpl.ts

```diff
@@ -602,6 +662,17 @@ export class CodeModelCliImpl implements CodeModelAz
         return this.codeModel.operationGroups[this.currentOperationGroupIndex].operations[this.currentMethodIndex].request.parameters[this.currentParameterIndex].language['python'].name;
     }
 
+    public get MethodParameter_NamePython(): string
```

Comment:
shouldn't it be done by python namer?

Reply:
No,  let's say an object A has a property B, cli wants B to be referred as `ToSnakeCase(A) + "_" + ToSnakeCase(B)`, python namer only returns `ToSnakeCase(B)`.